packages/aragon-api-react/package.json

```diff
@@ -17,7 +17,7 @@
     "access": "public"
   },
   "scripts": {
-    "build": "babel src --out-dir dist --source-maps",
+    "build": "babel src --out-dir dist --source-maps && terser dist/index.js --mangle --mangle-props --source-map --output dist/index.js",
```

Comment:
Not sure if we _really_ need to do this; are you afraid someone will be using this without a build step themselves?

Reply:
You’re right, I was optimizing and forgot that we don’t compress our libraries: I’ll remove it.